src/components/History/History.js

```diff
@@ -0,0 +1,55 @@
+import { useData } from "../../dataContext/DataContext";
+import { videos } from "../../data";
+import { Link } from "react-router-dom";
+import "../videoCard/videocard.css";
+
+export default function History() {
+  const { state } = useData();
+
+  let videosToDisplay = [];
+  videosToDisplay = getVideosToDisplay();
+  function getVideosToDisplay() {
+
+    state.history.forEach((videoId) => {
+      const historyVideo = videos.find(
+        (vi) => vi.id === videoId
+      );
+      videosToDisplay.push(historyVideo);
+      if (videosToDisplay.length === state.history.length) {
+        return videosToDisplay;
+      }
+    });
+    return videosToDisplay.reverse();
+  }
+  
+
+  return (
+    <>
+      <h2  style={{padding:"0.5rem",letterSpacing:"3px",fontSize:"1rem"}}>Recent Videos</h2>
```

Comment:
Avoid inline CSS

Reply:
This is used only for history page heading so it's better to use inline CSS.